butte/src/parser.rs

```diff
@@ -876,7 +925,7 @@ rpc_service Greeter {
         let result = rpc_decl(input);
         let expected = rpc!(
             Greeter,
-            doc!("A greeter service!"),
+            doc!(" A greeter service!"),
```

Comment:
Is this space addition intentional?

Reply:
Yeah, to make the comments appear exactly as they are written. Previously I was `str::trim`ing them.

Reply:
Got it 👍